src/cpu/pred/btb/ras.cc

```
@@ -121,6 +121,19 @@ BTBRAS::getPredictionMeta()
     return meta;
 }
 
+void
+BTBRAS::refreshPredictionMeta(Addr startAddr,
+                              const boost::dynamic_bitset<> &history,
+                              FullBTBPrediction &pred)
+{
+    (void)startAddr;
+    (void)history;
+    (void)pred;
+
```

Cppcheck (2.20.0)

[error] Reference to temporary returned.

(returnTempReference)